library/java/net/openid/appauth/AuthorizationException.java

```diff
@@ -559,6 +563,7 @@ public static AuthorizationException fromJson(@NonNull JSONObject json) throws J
                 JsonUtil.getStringIfDefined(json, KEY_ERROR),
                 JsonUtil.getStringIfDefined(json, KEY_ERROR_DESCRIPTION),
                 JsonUtil.getUriIfDefined(json, KEY_ERROR_URI),
+                json,
```

Comment:
```suggestion
                JsonUtil.getJsonObjectIfDefined(json, KEY_RESPONSE_JSON),
```

And toJson is missing the reponseJson field serialization:
```java
// Line 663
JsonUtil.putIfNotNull(json, KEY_ERROR_URI, errorUri);
JsonUtil.putIfNotNull(json, KEY_RESPONSE_JSON, responseJson);
return json;

```